pg_mon.c

```diff
@@ -334,22 +359,6 @@ _PG_init(void)
 	    ProcessUtility_hook = PU_hook;
 }
 
-/*
- * Module unload callback
- */
-void
-_PG_fini(void)
```

Comment:
see [ab02d702ef08343fba](https://git.postgresql.org/gitweb/?p=postgresql.git;a=commit;h=ab02d702ef08343fba30d90fdf7df5950063e8c9) for details